### PR TITLE
Added indian milk based beverage

### PR DESCRIPTION
**Pull request by realsidg**

### CODE REVIEW

**Review by alfg — CHANGES_REQUESTED** [review]
Image should be added to this PR.

**Comment by realsidg** [comment]
The image is added

File: src/recipes/doodh_cola.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            "ingredient": "Ice Cubes"
+        }
+    ],
+    "directions": [
+        "Take a large glass and add milk and ice cube.",
+        "Shake for 4 to 5 time.",
+        "Then add chilled Cola and again shake another few times.",
+        "Serve chilled Doodh Cola."
+    ],
+    "image": "https://1.bp.blogspot.com/-j5ZDuglbNAM/TgCro3Hj-hI/AAAAAAAAAo4/pUMykxbPrvo/s320/DSC01526.JPG",
```
Comment (alfg) [inline src/recipes/doodh_cola.json:28]:
```suggestion
    "image": "doodh-cola.jpg",
```

**Review by alfg — CHANGES_REQUESTED** [review]
Please update replace underscores with hyphens in filenames.

File: src/recipes/doodh_cola.json (hunk 1 above)
Comment (realsidg) [reply src/recipes/doodh_cola.json]: I made the suggested changes

**Comment by alfg** [comment]
Thanks!

**Pull request merged**